sql/core/src/main/java/org/apache/spark/sql/sources/v2/reader/Distribution.java

```diff
@@ -21,9 +21,9 @@
 
 /**
  * An interface to represent data distribution requirement, which specifies how the records should
- * be distributed among the {@link ReadTask}s that are returned by
- * {@link DataSourceV2Reader#createReadTasks()}. Note that this interface has nothing to do with
- * the data ordering inside one partition(the output records of a single {@link ReadTask}).
+ * be distributed among the {@link DataReaderFactory}s that are returned by
```

Comment:
`distributed among the data partition(one DataReader outputs data for one partition).`